package.json

```diff
@@ -34,7 +34,7 @@
     "babel-preset-es2015": "^6.13.2",
     "chai": "^3.5.0",
     "coveralls": "~2.11.1",
-    "dockerode": "^2.4.3",
```

Comment:
I think, this is not necessary anymore, since I dropped node 0.12 support alltogether.